guides/common/modules/proc_renewing-a-custom-ssl-certificate-on-smart-proxy.adoc

```diff
@@ -0,0 +1,66 @@
+[id="Renewing_a_Custom_SSL_Certificate_{smart-proxy-context}_{context}"]
+= Renewing a {customssltitle} certificate on {SmartProxyServer}
+
+[NOTE]
+====
+If your {SmartProxyServer} is already configured with CA-signed SSL certificates, and it is renewed by the same Certificate Authority, avoid replacing the consumer package `katello-ca-consumer-latest.noarch.rpm` on clients registered to {SmartProxyServer}.
+
+If the validity of CA-signed SSL certificates installed on {SmartProxy} from Certificate Authority A expires, and you installed an SSL certificate from Certificate Authority B, you must re-deploy the `katello-ca-consumer-latest.noarch.rpm` package on every client registered to external {SmartProxyServers}.
+You need a new CA certificate for a client to authenticate or communicate with {SmartProxyServers}.
+====
+
+Use this procedure to renew a {customssl} certificate signed by a Certificate Authority for your {SmartProxyServer}.
+The `{foreman-installer}` command, which the `{certs-generate}` command returns, is unique to each {SmartProxyServer}.
+You cannot use the same command on more than one {SmartProxyServer}.
+
+.Prerequisite
+You must create a new Certificate Signing Request and send it to the Certificate Authority to sign the certificate.
+In return, you will receive the {SmartProxyServer} certificate and CA bundle.
+You must keep the {SmartProxyServer} SSL certificates on the {ProjectServer} by creating a directory and verifying them by the `katello-certs-check` command.
+
+.Procedure
+. On {ProjectServer}, validate the {customssl} certificate input files:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# katello-certs-check -t {smart-proxy-context}\
+-b /root/_{smart-proxy-context}_cert/ca_cert_bundle.pem_ \
+-c /root/_{smart-proxy-context}_cert/{smart-proxy-context}_cert.pem_ \
+-k /root/_{smart-proxy-context}_cert/{smart-proxy-context}_cert_key.pem_ \
+----
+
+. Use the following `{certs-generate}` commands to generate the certificate archive file for your {SmartProxyServer}:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+{certs-generate} --foreman-proxy-fqdn "{smartproxy-example-com}" \
+--certs-tar  "_/root/My_Certificates/{smartproxy-example-com}-certs.tar_" \
+--server-cert "_/root/My_Certificates/{smart-proxy-context}_cert.pem_" \
+--server-key "_/root/My_Certificates/{smart-proxy-context}_cert_key.pem_" \
+--server-ca-cert "_/root/My_Certificates/ca_cert_bundle.pem_" \
+--certs-update-server
+----
+. On {ProjectServer}, copy the certificate archive file to your {SmartProxyServer}:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# scp /root/_{smart-proxy-context}_cert/{smartproxy-example-com}-certs.tar_
```

Comment:
This should match with the argument to `--certs-tar` in the above command. There you use `/root/My_Certificates/{smartproxy-example-com}-certs.tar`. You also lack an argument to `scp`, which is the destination.

Typically it's something like `scp source destination`, where destination is `user@{smartproxy-example-com}:_/root/{smartproxy-example-com}-certs.tar_`. Note that the path on the destination should match `-foreman-proxy-content-certs-tar` in the next command.

Reply:
I guess the destination should be `user@{smartproxy-example-com}:/root/My_Certificates`